app/src/main/java/at/aau/serg/activities/MainActivity.kt

```diff
@@ -19,6 +29,56 @@ class MainActivity : AppCompatActivity() {
             v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
             insets
         }
+
```

Comment:
Is it possible to move this code somewhere else so the Main activity is not crowded?

Reply:
yes, it is possible.

I have now added a class CallbackCreator, which now creates the callback object for every Request